support/routes/v2/formsg/formsgSiteLaunch.ts

```diff
@@ -252,6 +252,7 @@ export class FormsgSiteLaunchRouter {
         logger.info(
           `Domain ${launchResult.primaryDomainSource} does not have any AAAA records.`
         )
+        return [] // no AAAA records found
       }
       logger.error(
         `Error when trying to get AAAA records for domain ${launchResult.primaryDomainSource}: ${e}`
```

Comment:
/nit since you're editing this file, you could take the opportunity to clean up the neighbouring log statements. The existing statements does not print the error (since errors are not serializable).

```suggestion
      logger.error({
        message: 'Error when trying to get AAAA records for domain',
        error: e,
        meta: {
            domain: launchResult.primaryDomainSource
        }
      })
```

Same can be done for the other error() and info() logs seen as context code in the PR. 

But yeah, so main message with enahnced logging:
- report errors as-is, let the logger figure out how to export it
- log parameters as fields rather than interpolated content in the message

Reply:
since merged will create a separate pr ya

Reply:
Yes Sir 👍 . Thank you 🙏 .